bc_obps/service/tests/test_operation_service_v2.py

```diff
@@ -209,14 +179,17 @@ def test_update_operation_status_success():
         assert datetime.now(ZoneInfo("UTC")).replace(microsecond=0) - updated_operation.submission_date.replace(
             microsecond=0
         ) < timedelta(seconds=2)
+        assert updated_operation.registration_purpose == Operation.Purposes.OPTED_IN_OPERATION
```

Comment:
Seems fine to check this, but I'm not sure we need to since it's only added to make the mock data happy

Reply:
I threw it in to make sure that the operation's reg purpose wasn't getting overwritten somehow when the operation's status update happens (I'll admit that's very unlikely to happen). Happy to remove this line if you prefer.